src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/on_app_interface_unregistered_notification_test.cc

```diff
@@ -61,16 +61,20 @@ class OnAppInterfaceUnregisteredNotificationTest
     : public CommandsTest<CommandsTestMocks::kIsNice> {
  public:
   OnAppInterfaceUnregisteredNotificationTest()
-      : message_(CreateMessage())
-      , command_(
-            CreateCommand<OnAppInterfaceUnregisteredNotification>(message_))
-      , mock_app_(CreateMockApp()) {
+      : message_(CreateMessage()), command_(), mock_app_(CreateMockApp()) {
```

Comment:
@mvorobio wrong coding style

Reply:
@AByzhynar It is a result of Clang format with SDL style.

Reply:
@mvorobio Looks that you have the wrong setup of clang formatter. Return the way it was before.